packages/block-library/src/cover/save.js

```diff
@@ -124,42 +127,48 @@ export default function save( { attributes } ) {
 				style={ bgStyle }
 			/>
 
-			{ ! useFeaturedImage &&
-				isImageBackground &&
-				url &&
-				( isImgElement ? (
-					<img
-						className={ imgClasses }
-						alt={ alt }
+			{ /* @todo: added background wrapper in order to avoid repeating shared props between background types, test compatibility */ }
```

Comment:
I think it is best to separate cover block into new PR, as it needs to be tested well against all other properties both in global styles and block styles because of the wrapper.